libs/application/templates/driving-license/src/fields/EligibilitySummary/ReviewSection/index.tsx

```diff
@@ -29,10 +33,15 @@ type ReviewSectionProps = {
 
 const ReviewSection: FC<React.PropsWithChildren<ReviewSectionProps>> = ({
   application,
-  step: { state, description, title },
+  step: { state, description, title, daysOfResidency },
 }) => {
   const { formatMessage } = useLocale()
 
+  const showLocalRequirementDays: boolean =
+    isNumber(daysOfResidency) &&
```

Comment:
is it possible for the `daysOfResidency` to not be a number?

Reply:
had to use isNumber because it was rendering the number somehow, so this was the best solution